server/command-line/install.ts

```diff
@@ -47,9 +47,13 @@ program
 				.readFile(path.join(packageName.substring("file:".length), "package.json"), "utf-8")
 				.then((data) => JSON.parse(data) as typeof packageJson);
 		} else {
-			const split = packageName.split("@");
-			packageName = split[0];
-			const packageVersion = split[1] || "latest";
+			let packageVersion = "latest";
+			const atIndex = packageName.indexOf("@", 1);
+
+			if (atIndex !== -1) {
```

Comment:
Can you add a human friendly comment saying why you do that dance? It seems obvious now but give yourself a month or two ;)

Reply:
Something like how it is now?

Reply:
should work yes. I'd have given an example personally but that's bike shedding it to death.
Thanks.